modules/external-api/src/main/java/org/opencastproject/external/endpoint/EventsEndpoint.java

```diff
@@ -1704,17 +1704,22 @@ public Response getEventScheduling(@HeaderParam("Accept") String acceptHeader, @
   @Path("{eventId}/scheduling")
   @Produces({ ApiMediaType.JSON, ApiMediaType.VERSION_1_1_0, ApiMediaType.VERSION_1_2_0 })
   @RestQuery(name = "updateeventscheduling", description = "Update an event's scheduling information.", returnDescription = "", pathParameters = {
-      @RestParameter(name = "eventId", description = "The event id", isRequired = true, type = STRING) }, restParameters = {
-      @RestParameter(name = "scheduling", isRequired = true, description = "Scheduling Information", type = STRING) }, reponses = {
+      @RestParameter(name = "eventId", description = "The event id", isRequired = true, type = Type.STRING) }, restParameters = {
+      @RestParameter(name = "scheduling", isRequired = true, description = "Scheduling Information", type = Type.STRING),
+      @RestParameter(name = "allowConflict", description = "Allow conflicts when updating scheduling", defaultValue = "false", isRequired = false, type = Type.BOOLEAN) }, reponses = {
```

Comment:
Due to MH-13416 REST docs defaultValue not correctly set for boolean parameters, setting the defaultValue for a boolean REST parameter will cause the REST docs to always check the checkbox representing the value of this parameter. As far now, the overall result is better when you just omit the defaultValue (will default to false in that case)

Reply:
Ok, changing this and it's good to know.